am/README.md

```diff
@@ -333,6 +333,13 @@ httpClient:
 | ui.service.internalPortName | string | `http` |  |
 | ui.service.name | string | `"nginx"` |  |
 | ui.service.type | string | `"ClusterIP"` |  |
+| userManagement.activity.enabled | boolean | `false` | |
```

Comment:
options you added in the .adoc file to enable the connector are missing in this one

Reply:
Will add them